views/css/adyen.css

```diff
@@ -143,3 +143,13 @@ body#checkout section.checkout-step .payment-options label {
 .adyen-payment .adyen-collapser.adyen-payment-method-label {
     cursor: pointer;
 }
+
+@media only screen and (max-width: 480px) {
+    body#checkout section.checkout-step .payment-options .custom-radio {
+        margin-right: 0;
+    }
+
+    .payment-option label {
+        text-align: left;
+    }
```

Comment:
Could these CSS rules have an 'adyen' in the selector so that we're sure that it is going to apply only to our elements?

Reply:
The rendering of the label and logo are done trough the default Presta functions (`setCallToActionText` & `setLogo`) so I'm not sure how we would add a custom class to them, whereas adding a class to the form would have been straightforward since we have the smarty template files for them.

Also I think that since we added the config value to disable the adyen styling in #208, we are already providing them with the ability to opt out of our styling.